style.css

```diff
@@ -0,0 +1,317 @@
+@import 'https://fonts.googleapis.com/css?family=Lato:400,700';
+
+#body {
+  background-color: #eee;
+  font-family: 'Lato', sans-serif;
+}
+
+#page-wrapper {
+  position: relative;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  color: #000;
+  text-decoration: none;
+}
+
+.container {
+  max-width: 1000px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.nav-link {}
+
+i1 {
+  font-size: 40px;
+  font-weight: 900;
+  font-family: Bakery;
+}
+
+
+header { 
+  position: fixed; 
+  top: 0; 
+  min-height: 75px; 
+  padding: 0px 20px; 
+  display: flex; 
+  justify-content: space-around; 
+  align-items: center; 
+  background-color: #eee;
+}
+
+@media (max-width: 600px) {
+  header {
+    flex-wrap: wrap;
+  }
+}
+
+#features {
+  margin-top: 30px;
+}
+
+
+#features .icon { 
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 125px;
+  width: 20vw;
+  color: darkorange;
+  
+} 
+
+@media (max-width: 550px) {
+  #features .icon {
+    display: none;
+  }
+}
+
+#features .desc { 
+  display: flex; 
+  flex-direction: column; 
+  justify-content: center; 
+  height: 125px; 
+  width: 80vw; 
+  padding: 5px; 
+} 
+
+@media (max-width: 550px) { 
+  #features .desc { 
+    width: 100%; 
+    text-align: center; 
+    padding: 0; 
+    height: 150px; 
+  } 
+} 
+
+@media (max-width: 650px) { 
+  #features { 
+    margin-top: 0; 
+  } 
+} 
+
+#how-it-works { 
+  margin-top: 50px; 
+  display: flex; 
```

Comment:
.you use to many @media query. I think it would be better to use few media query 😀